cocos/2d/CCGrid.h

```diff
@@ -203,55 +252,71 @@ class CC_DLL Grid3D : public GridBase
     bool _oldDepthWriteValue;
 };
 
-/**
- TiledGrid3D is a 3D grid implementation. It differs from Grid3D in that
- the tiles can be separated from the grid.
+/**@~english
+ * TiledGrid3D is a 3D grid implementation. It differs from Grid3D in that
+ * the tiles can be separated from the grid.
+ * @~chinese 
+ * TiledGrid3D是一个3D网格实现。它不同于Grid3D,
+ * 每一个瓦片(tile)可以独立于网格。
 */
 class CC_DLL TiledGrid3D : public GridBase
 {
 public:
-    /** Create one Grid. */
+    /** @~english Create one Grid.  @~chinese 创建一个网格。*/
     static TiledGrid3D* create(const Size& gridSize, Texture2D *texture, bool flipped);
-    /** Create one Grid. */
+    /** @~english Create one Grid.  @~chinese 创建一个网格。*/
     static TiledGrid3D* create(const Size& gridSize);
-    /**
-     Constructor.
+    /**@~english
+     * Constructor.
+     * @~chinese 
+     * 构造函数。
      * @js ctor
      */
     TiledGrid3D();
-    /**
-     Destructor.
+    /**@~english
+     * Destructor.
+     * @~chinese 
+     * 析构函数。
      * @js NA
      * @lua NA
      */
     ~TiledGrid3D();
 
-    /** Returns the tile at the given position.
+    /** @~english Returns the tile at the given position.
+     * @~chinese 返回给定位置的瓦片。
      * @js NA
      * @lua NA
      */
     Quad3 getTile(const Vec2& pos) const;
-    /** returns the tile at the given position 
+    /** @~english returns the tile at the given position 
+     * @~chinese 返回给定位置的瓦片.
+     * @deprecated Use getTile instead.
```

Comment:
@deprecated tag don't need to be translated, but deprecated functions should be documented too